README.md

```diff
@@ -148,23 +148,24 @@ The functions will be loaded in the same order as they are inside the array.
 -------------------------------------------------------
 <a name="after"></a>
 
-### app.after(func([context], [done]), [cb])
+### app.after(func(error, [context], [done]), [cb])
```

Comment:
Agree, but at the moment this is the behaviour:
```js
test('error', (t) => {
  t.plan(2)

  const server = { my: 'server' }
  const app = boot(server)

  app.use(function (s, opts, done) {
    done(new Error('err'))
  }, err => {
    t.ok(err)
    app.use(function (s, opts, done) {
      done()
    })
  })

  app.ready(function (err) {
    t.ok(err)
  })
})
```
As you can see, we have not that capability, because the error will arrive to the ready callback in every case.
With after I can "block" that error ina  specific point, as you can see here:
```js
test('error', (t) => {
  t.plan(3)

  const server = { my: 'server' }
  const app = boot(server)

  app.use(function (s, opts, done) {
    done(new Error('err'))
  }, err => {
    t.ok(err)
  })

  app.after(function (err) {
    t.ok(err)
    app.use(function (s, opts, done) {
      done()
    })
  })

  app.ready(function (err) {
    t.error(err)
  })
})
```
This is why yesterday I was saying that we should raise the error in the ready callback only if the error is not handled in the use callback.